iphone/TitaniumKit/TitaniumKit/Sources/API/KrollBridge.m

```diff
@@ -314,9 +314,8 @@ - (void)evalFileOnThread:(NSString *)path context:(KrollContext *)context_
     }
   }
   if (exception != NULL) {
-    id excm = [KrollObject toID:context value:exception];
     evaluationError = YES;
-    [[TiExceptionHandler defaultExceptionHandler] reportScriptError:[TiUtils scriptErrorValue:excm]];
+    [TiExceptionHandler.defaultExceptionHandler reportScriptError:exception inKrollContext:context];
```

Comment:
@janvennemann  defaultExceptionHandler is a method not property. I think it should be used as  [TiExceptionHandler defaultExceptionHandler]

Reply:
Technically, a property will only create a variable and getter/setters for you and the compiler will automatically call the getter method when you access the property. By using `TiExceptionHandler.defaultExceptionHandler` here I just instruct the compiler to automatically call the method `defaultExceptionHandler`, so it shouldn't make a difference.